rct229/rules/section5.py

```diff
@@ -110,6 +116,114 @@ def rule_check(self, context, calc_vals, data=None):
 # ------------------------
 
 
+class Section5Rule5(RuleDefinitionListIndexedBase):
+    """Rule 5 of ASHRAE 90.1-2019 Appendix G Section 5 (Envelope)"""
+
+    def __init__(self):
+        super(Section5Rule5, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            required_fields={
+                "weather": ["climate_zone"],
+            },
+            each_rule=Section5Rule3.BuildingRule(),
```

Comment:
@charliepnnl  Revise it to Section5Rule5.BuildingRule()

Reply:
Done